schema/GetOrder.json

```diff
@@ -1,685 +1,690 @@
-                      "type": "number"
-                    },
-                    "is_virtual": {
-                      "type": "number"
-                    },
-                    "item_id": {
-                      "type": "number"
-                    },
-                    "name": {
-                      "type": "string"
-                    },
-                    "no_discount": {
-                      "type": "number"
-                    },
-                    "order_id": {
-                      "type": "number"
-                    },
-                    "original_price": {
-                      "type": "number"
-                    },
-                    "price": {
-                      "type": "number"
-                    },
-                    "price_incl_tax": {
-                      "type": "number"
-                    },
-                    "product_id": {
-                      "type": "number"
-                    },
-                    "product_type": {
-                      "type": "string"
-                    },
-                    "qty_canceled": {
-                      "type": "number"
-                    },
-                    "qty_invoiced": {
-                      "type": "number"
-                    },
-                    "qty_ordered": {
-                      "type": "number"
-                    },
-                    "qty_refunded": {
-                      "type": "number"
-                    },
-                    "qty_shipped": {
-                      "type": "number"
-                    },
-                    "quote_item_id": {
-                      "type": "number"
-                    },
-                    "row_invoiced": {
-                      "type": "number"
-                    },
-                    "row_total": {
-                      "type": "number"
-                    },
-                    "row_total_incl_tax": {
-                      "type": "number"
-                    },
-                    "row_weight": {
-                      "type": "number"
-                    },
-                    "sku": {
-                      "type": "string"
-                    },
-                    "store_id": {
-                      "type": "number"
-                    },
-                    "tax_amount": {
-                      "type": "number"
-                    },
-                    "tax_invoiced": {
-                      "type": "number"
-                    },
-                    "tax_percent": {
-                      "type": "number"
-                    },
-                    "updated_at": {
-                      "type": "string"
-                    },
-                    "weight": {
-                      "type": "number"
-                    }
-                  }
-                }
               }
             }
           }
         }
       }
     }
-  }
+  },
+  "tags": ["Get", "Order", "SalesOrder"],
```

Comment:
I think this should just be 
"tags": ["Get", "SalesOrder"],

Reply:
PR has been updated with the suggested change